crates/abq_cli/src/workers.rs

```diff
@@ -161,6 +173,18 @@ async fn do_shutdown(
     suite_result
         .write_short_summary_lines(&mut stdout, ShortSummaryGrouping::Runner)
         .unwrap();
+    print!("\n\n");
+    if execution_mode == ExecutionMode::WriteNormal {
+        print!("Run the following command to replay these tests locally:");
```

Comment:
Do you think we should include a message about setting RWX_ACCESS_TOKEN or running `abq login`?